torch/_tensor.py

```diff
@@ -358,6 +358,7 @@ def _reduce_ex_internal(self, proto):
                 self.stride(),
                 self.requires_grad,
                 backward_hooks,
+                torch._utils.get_math_bits(self),
```

Comment:
You can't do this. This will break serialization FC.

What you need to do is test if get_math_bits is the default value (all not set). If none of it is set, then you must omit it from the argument list, so that it is compatible with old rebuild_tensor.

Reply:
Make sense. Thanks!